OpenRA.Game/Network/Session.cs

```diff
@@ -102,7 +102,7 @@ public static Client Deserialize(MiniYaml data)
 			public int Index;
 			public HSLColor PreferredColor; // Color that the client normally uses from settings.yaml.
 			public HSLColor Color; // Actual color that the client is using. Usually the same as PreferredColor but can be different on maps with locked colors.
-			public string Country;
+			public string CountryRace;
```

Comment:
`Country` to `Faction` please.
You are renaming this anyway, so might as well do it right the first time.